README.md

```diff
@@ -14,7 +14,7 @@ srs-enb-ue:
   enb-mcc: <mcc>
   enb-mnc: <mnc default>
   enb-rf-device-name: <zmq>
-  enb-rf-device-args: <RF Device Name.>
+  enb-rf-device-args: <RF Device Name.> #for example: "fail_on_disconnect=true,tx_port=tcp://*:2000,rx_port=tcp://localhost:2001,id=enb,base_srate=23.04e6"
```

Comment:
How about setting that example as the default value and removing it from the `README`?

Reply:
We might want to specify here in the `README` which fields are optional and which ones aren't. What do you think?

Reply:
I agree, fixed :)